### PR TITLE
chore: add script for polygon vault deprecation

### PR DESCRIPTION
**Pull request by axejintao**
# Summary

- deprecates remaining 3 polygon vaults
- solve #729

### CODE REVIEW

**Review by sajanrajdev — DISMISSED** [review]
Pushed a change to the Badger.py library so that Eth specific contracts are only initialized when running on ETH, getting an error on Polygon otherwise.

Deprecated the three active vaults on Polygon and confirms this.

LGTM

For other reviewers, run with: 
```
brownie run scripts/issue/729/deprecate_polygon.py deprecate_polygon --network polygon-main-fork
```

**Comment by gosuto-inzasheru** [comment]
i dont see a need for 822a077 @sajanrajdev. i believe the fact that we started using `r` from the registry instead of `registry.<chain>` in for example `badger.py` should be enough. `GreatApeSafe.contract` just returns `None` in the case of an invalid contract. how did that break anything on your side?

File: scripts/issue/729/deprecate_polygon.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+TECH_OPS = registry.poly.badger_wallets.techops_multisig
+
+REGISTRY = registry.poly.registry_v2
+
+WBTC_IBBTC_SLP = registry.poly.sett_vaults.bslpibBTCWbtc
+WBTC_USDC_QLP = registry.poly.sett_vaults.bqlpUsdcWbtc
+AWBTC_REN_CRV = registry.poly.sett_vaults.bcrvRenBTC
+
+def deprecate_polygon():
```
Comment (gosuto-inzasheru) [inline scripts/issue/729/deprecate_polygon.py:12]: would opt for using default `main` here if there is no good reason not to. this saves having to specify the func name when calling the script:
```
brownie run issue/729/deprecate_polygon deprecate_polygon
```
vs
```
brownie run issue/729/deprecate_polygon
```

**Review by gosuto-inzasheru — APPROVED** [review]
runs fine, also with `HEAD` at ee470e8.

would be nice to get rid of your commit that follows that one actually @sajanrajdev, unless there is something im missing?

**Comment by sajanrajdev** [comment]
> i dont see a need for [822a077](https://github.com/Badger-Finance/badger-multisig/commit/822a077bbacb10880482de17063e26a93cce6729) @sajanrajdev. i believe the fact that we started using `r` from the registry instead of `registry.<chain>` in for example `badger.py` should be enough. `GreatApeSafe.contract` just returns `None` in the case of an invalid contract. how did that break anything on your side?

@gosuto-inzasheru, Interesting, will remove my commit and dig into what is breaking on my side but you are right, it should be enough.

**Review by gosuto-inzasheru — APPROVED** [review]

**Review by sajanrajdev — APPROVED** [review]
Script LGTM and was able to run it properly to confirm with the reverted changes. I continue to have issues with it but it is likely due to a local brownie mis config or rpc. 

Approving since it runs properly you for you as it is.

**Pull request merged**